neural_compressor/torch/algorithms/weight_only/__init__.py

```diff
@@ -15,3 +15,6 @@
 from .utility import *
 from .rtn import rtn_quantize
 from .gptq import gptq_quantize
+
+from .hqq.quantizer import HQQuantizer
+from .hqq.config import HQQModuleConfig, QTensorConfig
```

Comment:
How about we unify the usage as rtn, gptq? a `hqq_quantize` interface

Reply:
Thanks, aligned the API in [0623ab3](https://github.com/intel/neural-compressor/pull/1597/commits/0623ab34068aa88e63b3f153c477becd025f174b).